scholia/app/templates/venue.html

```diff
@@ -240,6 +240,14 @@
  ORDER BY DESC(?count)
  `
 
+ authorAwardsSparql = `
+SELECT DISTINCT ?author ?authorLabel ?award ?awardLabel WHERE {
+  ?item wdt:P1433 wd:{{ q }} ;
+        wdt:P50 ?author .
+  ?author wdt:P166 ?award .
+SERVICE wikibase:label { bd:serviceParam wikibase:language "[AUTO_LANGUAGE],en". }  
```

Comment:
I extended this with some more languages. Maybe the service labeling should be parametrized.